lib/src/package_analyzer.dart

```diff
@@ -28,12 +28,14 @@ import 'utils.dart';
 class InspectOptions {
   final String? pubHostedUrl;
   final String? dartdocOutputDir;
+  final Duration? totalTimeBudget;
   final Duration? dartdocTimeout;
   final int? lineLength;
 
   InspectOptions({
     this.pubHostedUrl,
     this.dartdocOutputDir,
+    this.totalTimeBudget,
```

Comment:
Could we document this a bit.
Can you pass both timeouts? How do they interact? Are there defaults?

Reply:
Updated this and other options. PTAL